source/brailleDisplayDrivers/alva.py

```diff
@@ -142,7 +142,10 @@ def _updateSettings(self):
 			displaySettings = self._dev.getFeature(ALVA_DISPLAY_SETTINGS_REPORT)
 			self.numCells = ord(displaySettings[ALVA_DISPLAY_SETTINGS_CELL_COUNT_POS])
 			timeStr = self._dev.getFeature(ALVA_RTC_REPORT)[1:ALVA_RTC_STR_LENGTH+1]

```

Comment:
Note that this part actually never caused an issue. Therefore my initial placement of this line in a try/except block was unnecessary